config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
-resources :users, only: [:index, :show] do
-  resources :posts, only: [:index, :show]
-end
-end
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:new, :create]
+        resources :likes, only: [:new, :create]
+    end
+  end
```

Comment:
- [Optional] Dear @felixDev22, It would be wise to have a root path (or some other brought) for user to create an account. currently I have to navigate to the terminal to creat  new user.

Reply:
Thank you for approving 😄 and giving wise advice 👍